lib/versions_repository.py

```diff
@@ -25,3 +25,21 @@ def setup_versions_repository(config):
         raise
 
     return versions_repo
+
+def read_version_and_milestone(versions_repo):
+    """
+    Read current version and milestone (alpha or beta) from VERSION file
+
+    Args:
+        versions_repo (GitRepository): packages metadata git repository
+    Returns:
+        version_milestone (str): version and milestone
```

Comment:
Pls document the expected format of this string (`<version`>-`<milestone`>, valid milestone values: alpha, beta)